### PR TITLE
Copyable

### PR DESCRIPTION
**Pull request by nickmccurdy**
Depends on #10. ~Open to review (ignoring the Tooltip files). Do not merge yet.~

image

### CODE REVIEW

**Comment by Sonicrida** [comment]
Is `Tooltip` duplicated here since #10 hasn't been merged yet?

**Comment by Sonicrida** [comment]
Never mind I missed the "(ignoring the Tooltip files" 😂

File: src/atoms/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+const InnerTooltip = styled.div<{ height?: number }>`
+  border: 0.0625em solid #bac9d4;
+  background: ${props => props.theme.controlBackground};
+  border-radius: ${borderRadius};
+  box-shadow: 0 0.1875em 0.375em 0 #00000012,
+    0 0.4375em 0.625em 0.4375em #32325d1a;
+  ${padding(scale(-1))};
+  position: absolute;
+  bottom: ${props => props.height}px;
```
Comment (Sonicrida) [inline src/atoms/Tooltip/Tooltip.tsx]: Should we use `em` here?

Comment (Sonicrida) [reply src/atoms/Tooltip/Tooltip.tsx]: Alright! I was wondering about if that would be an issue so that makes sense. Thanks Nick!

**Comment by nickmccurdy** [comment]
Ready for review after #10

**Comment by nickmccurdy** [comment]
Needs design review

**Comment by nickmccurdy** [comment]
Arrow needs to be centered on children, not tooltip

**Comment by nickmccurdy** [comment]
This positioning issue is a lot of work to fix and only affects larger sizes of tooltips. I'll merge and add this bug to the backlog.

**Pull request merged**